src/models/ProjectNotification.ts

```diff
@@ -57,7 +57,18 @@ export class ProjectNotification {
 
         const { displayName: user, username } = this.currentUser.data;
         const subject = i18n.t("{{username}} is requesting a data review", { username });
-        const pageLink = window.location.href;
+
+        const pageOrigin = window.location.origin;
+        const projectId = this.project.id;
+        const dataApprovalLink =
+            pageOrigin +
+            "/api/apps/Data-Management-App/index.html#/data-approval/" +
+            projectId +
+            "/" +
+            dataSetType +
+            "/" +
+            period;
+
```

Comment:
We have `getUrl` as an example of how to generate programmatically the route and make it work for DEV and APP (with features involving links, check always that the app works). I've sent you a proposal, check it and upload with the changes you see fit.